go.sum

```diff
@@ -15,6 +15,7 @@ github.com/stretchr/testify v1.6.1 h1:hDPOHmpOpP40lSULcqw7IrRb/u7w6RpDC9399XyoNd
 github.com/stretchr/testify v1.6.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
 golang.org/x/time v0.0.0-20201208040808-7e3f01d25324 h1:Hir2P/De0WpUhtrKGGjvSb2YxUgyZ7EFOSLIcSSpiwE=
 golang.org/x/time v0.0.0-20201208040808-7e3f01d25324/go.mod h1:tRJNPiyCQ0inRvYxbN9jk5I+vvW/OXSQhTDSoE431IQ=
+gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405 h1:yhCVgyC4o1eVCa2tZl7eS0r+SDo693bJlVdllGtEeKM=
```

Comment:
What is this change for?

Reply:
Sorry, I did even not notice this before.
I think it was added by `go mod tidy` formatting `go.sum`, and after deleting it and rerunning `go mod tidy`, it only added this line.
Should I delete the line and resubmit?

Reply:
> What is this change for?

Hello there, I deleted that line, committed with no other change and did a force-push. Could you please have a look again?

Thanks.